src/app/components/showcase.tsx

```diff
@@ -0,0 +1,15 @@
+export default function Showcase() {
+  return (
+    <div className="p-5 mt-[100vh] bg-black">
+      <h1 className="pb-3 text-center font-serif text-2xl">{"Photography"}</h1>
+      <h2 className="pb-3 font-serif text-xl">{"Sports"}</h2>
```

Comment:
You have a decent chunk of repeated code here. Could be an opportunity to create a new `ShowcaseItem` component (in this same file). Then you could do something like 
```suggestion
      <ShowcaseItem>Sports</ShowcaseItem>
```
or if you don't want to use `children`, you could do
```suggestion
      <ShowcaseItem title="Sports" />
```

Another option is including the h1 in this new component so you could do 
```suggestion
      <ShowcaseItem title="Photography" menuItems={['Sports', 'Automotive', 'Landscapes']} />
```

Reply:
Thanks! Yeah, I found a lot of repeat code with the utility classes that I wasn't used to, but I've since refactored these into <ShowcaseItem> components where the title is accepted as a prop, and the description is added as children. This has been updated in a recent commit